Core.PoseEditor/HSPE.cs

```diff
@@ -47,7 +47,7 @@ internal class HSPE : GenericPlugin
         public const string GUID = "com.joan6694.illusionplugins.poseeditor";
         internal const int saveVersion = 0;
 #endif
-        public const string Version = "2.16.0";
+        public const string Version = "2.16.1";
```

Comment:
```suggestion
        public const string Version = "2.16.0";
```
No version bumps in PRs pls.